src/main/java/com/codeofus/rent_a_park/repositories/PersonRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.codeofus.rent_a_park.repositories;
+
+import com.codeofus.rent_a_park.models.Person;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface PersonRepository extends JpaRepository<Person, Integer> {
+
+    Person getPersonByRegistration(String registration);
```

Comment:
What if there's no such registration, and is there a constraint that it's unique so that it returns at most one?